core/src/main/scala/cats/data/package.scala

```diff
@@ -1,9 +1,11 @@
 package cats
 
+import scala.collection.immutable.{Stream => SStream}
```

Comment:
This seems unnecessary since the cats version is called `Streaming` as opposed to `Stream`. I'm wondering if this is just here because you might have started out calling the cats version `Stream`?